src/object_list.c

```diff
@@ -473,14 +473,14 @@ void object_create_identifier_name(uint8* string_buffer, rct_object_entry* objec
 
 	for (uint8 i = 0; i < 4; ++i){
 		uint8 flag_part = (object->flags >> (i * 8)) & 0xFF;
-		*string_buffer++ = RCT2_ADDRESS(0x0098DA64, uint8)[flag_part >> 4];
-		*string_buffer++ = RCT2_ADDRESS(0x0098DA64, uint8)[flag_part & 0xF];
+		*string_buffer++ = RCT2_ADDRESS(0x0098DA64, char)[flag_part >> 4];
```

Comment:
Seems like a good reason to never use it - always prefer sint8 or uint8 or char8.